app/scripts/redirects.tsx

```diff
@@ -52,3 +55,8 @@ export const discoveryRoutes = (
     <Route path='discoveries/:discoveryId' element={<DiscoveryRedirect />} />
   </Route>
 );
+
+export function DatasetExploreRedirect() {
+  const dataset = useDataset();
+  return dataset?.data.disableExplore ? <DatasetsOverview /> : <DatasetsExplore />;
```

Comment:
I think this should be a redirect (`<Navigate replace ... />`) and not a conditional rendering. Otherwise the url `/explore` will render the overview.

---

For my curiosity: why have this check in an additional component, instead of the Explore page? Wouldn't it make more sense to look at the explore page code and see immediately what makes it behave as it does?